<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowCustomer</class>
 <widget class="QMainWindow" name="MainWindowCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Форма клиента</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Change_info_user">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
background-color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Редактировать профиль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 0, 40);
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>Выйти из профиля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
background-color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Корзина</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find_products">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(0, 228, 58)}
QPushButton{border: 2px solid black}</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_find_product">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{border: 1px solid black}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_find">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Введите критерии поиска:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выберите критерий сортировки:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_orders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
background-color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Заказы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_filters">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(248, 30, 20)}
QPushButton{border: 2px solid black}</string>
    </property>
    <property name="text">
     <string>Сбросить </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>1051</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>photo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>optprice</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>roznprice</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>description</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_sort_products">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215);
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_to_cart">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 153, 18);
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>Добавить в корзину</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
